core/src/main/java/com/linecorp/armeria/common/RequestHeaders.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Consumer;
+
+/**
+ * Immutable HTTP/2 headers for an {@link HttpRequest}.
+ *
+ * @see ResponseHeaders
+ */
+public interface RequestHeaders extends HttpHeaders, RequestHeaderGetters {
+
+    /**
+     * Creates a new empty builder.
```

Comment:
Guess this and below should be `Returns` for consistency with the other ones

Reply:
Fixed